src/ValidatedNumerics.jl

```diff
@@ -26,25 +22,31 @@ import Base:
     ⊆, eps,
     floor, ceil, trunc, sign, round,
     expm1, log1p,
-    isfinite, isnan
+    isfinite, isnan,
+    precision
+
+if VERSION >= v"0.5.0-dev+1182"
+    import Base: rounding, setrounding, setprecision
+else
+    import Compat: rounding, setrounding, setprecision
+end
```

Comment:
I don't quite understand why this is written like this. 

Isn't it enough to add `@compat` when it is needed, avoiding this `if`?